x-pack/plugin/security/src/test/java/org/elasticsearch/xpack/security/transport/ServerTransportFilterTests.java

```diff
@@ -48,7 +48,6 @@
 import static org.elasticsearch.xpack.core.security.support.Exceptions.authenticationError;
```

Comment:
We gotta fix up `testInboundDestructiveOperations` as well.

Reply:
Good catch  👍  I totally missed it